lib/govuk_index/presenters/parts_presenter.rb

```diff
@@ -0,0 +1,50 @@
+module GovukIndex
+  class PartsPresenter
+    def initialize(parts: [])
+      @parts = parts
+    end
+
+    def presented_parts
+      return unless parts && parts.any?
+
+      parts.map do |part|
+        {
+          "slug" => part["slug"],
+          "title" => part["title"],
+          "body" => summarise(part.fetch("body", [{}])),
```

Comment:
I still think we should have the full body in the search index, if we did before.  The `summarize` part could happen when presenting search results.

Reply:
As discussed offline we'll re-use the body field for the truncated body, and not add a summary field.